src/dao/MoodDao.java

```diff
@@ -1,26 +1,42 @@
 package dao;
 
 import foodmood.Mood;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
 import java.util.ArrayList;
+import java.util.List;
 
 /**
  * Provides methods for reading mood objects from the database.
  *
  * Write functionality is not provided because our requirements do not allow the
  * creation of new moods. However, this could be added in the future.
  *
- * @author jsm158
+ * @author matt cano
  *
  */
 public class MoodDao {
-    
-    private final ConnectionManager cm;
+
+////    private final ConnectionManager cm;
+    private static final String FILE_PATH = "data/Moods.csv";
+    private static final String COMMA_DELIMITER = ",";
+    private static final String NEW_LINE = "\n";
+    private static final String FILE_HEADER = "Id,MoodName";
+
+    private static final int MOOD_ID_INDEX = 0;
+    private static final int MOOD_NAME_INDEX = 1;
 
     /**
      * Constructs a new MoodDao
      */
-    public MoodDao() {
-        this.cm = new ConnectionManager();
+    public MoodDao() throws DaoException {
```

Comment:
Having this code in the constructor will cause the moods file to be written every time a MoodDao object is instantiated, which is suboptimal as it will overwrite the previous file. Since we're not letting users create moods this isn't a big deal, but it's not necessary.